src/Illuminate/Foundation/Http/MaintenanceModeBypassCookie.php

```diff
@@ -15,7 +15,7 @@ class MaintenanceModeBypassCookie
      */
     public static function create(string $key)
     {
-        $expiresAt = Carbon::now()->addHours(12);
+        $expiresAt = Carbon::now()->addHours(config('session.maintenance_bypass_cookie_lifetime'));
```

Comment:
According to your config doc block,
> number of minutes

this is now minutes that are added
```suggestion
        $expiresAt = Carbon::now()->addMinutes(config('session.maintenance_bypass_cookie_lifetime'));
```